services/hooks/src/v1.js

```diff
@@ -573,6 +594,16 @@ const triggerHookCommon = async function({req, res, hook, payload, clientId, fir
   }
 };
 
+const isDeniedBinding = async ({bindings, denyList}) => {
+  let denyPattern = new RegExp(`^exchange/(${denyList.join('|')})/`);
+  let denied=false;
+  bindings.forEach((binding) => {
```

Comment:
this could also be done with return [`!bindings.every(..)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) (but no need to change it).